src/hooks/useEditorEffect.ts

```diff
@@ -33,7 +33,9 @@ export function useEditorEffect(
   // every time it changes, because it will most likely
   // be defined inline and run on every re-render.
   useLayoutGroupEffect(
-    () => effect(editorView),
+    () => {
+      if (editorView) effect(editorView);
```

Comment:
we currently assume that this is never false, but what if it sometimes is for some unforeseen reason, what do we do? should we fail silently as we do here, warn, or throw?

Reply:
I think the answer to this is no, it's not a failure if `editorView` is undefined when this runs, it just means it's not ready to run the effect yet

Reply:
yeah that aligns with my thinking. great!